docs/source/migration.rst

```diff
@@ -3,7 +3,25 @@ Migration
 
 Some mango versions will break the API; in this case, we may provide instructions on the migration on this page.
 
+********************
+mango 1.2.X to 2.0.0
+********************
 
+* The methods send_acl_message and schedule_instant_acl_message have been removed, use send_message/schedule_instant_message with create_acl instead if you explicitly need an ACLMessage, otherwise you can just replace it with send_message/schedule_instant_message. 
+  * Background: In the past, mango relied on ACL messages for message routing, in 2.0.0 mango introduces an internal message container for all types of content, which provides necessary routing information. That way all mango messages can always be routed to the correct target. 
+* The container factory method create_container has been removed and instead there is one factory method per container type now: create_tcp_container, create_mqtt_container, create_external_container
+* The container factory methods are not async anymore! 
+* The startup for the TCP-server/MQTT-client has been moved to `container.start()`.
+  * Use the async context manager provided by `activate(containers)` as described in the documentation
+  * Background: This has the advantage that calling shutdown is not necessary anymore, in that way it is not possible to forget and ressource leaks are avoided at any time 
+* Consider to use the async context manager provided by run_with_X, these managers will provide suitable containers internally, that way the user does not need to handle container at all
+* Agents are created without container now
+  * This implies that Agents need to be registered explicitly using `register`
```

Comment:
Going by the new examples this should be `include` or `register` here? Clearly the `include` API can do this without me ever typing `register`.